ergo/platforms/metaculus/metaculus.py

```diff
@@ -183,7 +184,14 @@ def get_questions(
         questions_json = self.get_questions_json(
             question_status, player_status, cat, pages, False
         )
-        return [self.make_question_from_data(q) for q in questions_json]
+        questions = []
+        for q in questions_json:
+            try:
+                questions.append(self.make_question_from_data(q))
+            except NotImplementedError as e:
+                if not fail_silent:
+                    raise e
+        return questions
```

Comment:
I assume this is for log date questions that we're not handling yet? Would be nice if we could skip them more explicitly somehow.